.github/workflows/release-embedding-sdk.yml

```diff
@@ -65,14 +118,14 @@ jobs:
         run: yarn embedding-sdk:test-unit
 
   build-sdk:
-    needs: test
+    needs: [test, determine-version]
```

Comment:
We can only access direct job dependency, so to get the next SDK version number I need to depend on this new job.